README.rst

```diff
@@ -392,10 +392,76 @@ returned by the server in the specified time. For example:
 
 This will cause both add_event() and add_events() to timeout after 100 seconds. If this timeout limit is hit, a requests.Timeout will be raised. Due to a bug in the requests library, you might also see an SSLError (https://github.com/kennethreitz/requests/issues/1294)
 
-Create Scoped Keys
+Create Access Keys
 ''''''''''''''''''
 
-The Python client enables you to create `Scoped Keys <https://keen.io/docs/security/#scoped-key>`_ easily. For example:
+The Python client enables the creation and manipulation of `Access Keys <https://keen.io/docs/access/access-keys>`_. Examples:
+
+.. code-block:: python
+
+    import keen
```

Comment:
maybe `from keen.client import KeenClient` instead? 

I'm not tied to that way, but for consistency?

Reply:
Basic queries don't require the client to be used. Eventually I'd like to make it clear that you use the client object if you need to use different projects in the same python process, but otherwise you can just use the keen package to run your stuff.  If you think it would be clearer within the docs to use the Client (and make a mention of the root package being usable for this as well) I can make that change.